### PR TITLE
New version: WolframResearch.WolframEngine version 14.0.0

### PR DESCRIPTION
**Pull request by cc-wr**
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146500)

### CODE REVIEW

**Review by thmeyer-msft — APPROVED** [review]

File: manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+PackageVersion: 14.0.0
+MinimumOSVersion: 10.0.17763
+InstallerType: wix
+Installers:
+- Architecture: x64
+  InstallerUrl: http://files.wolframcdn.com/packages/winget/14.0.0.0/WolframEngine_14.0.0_WIN.msi
+  InstallerSha256: 97A67DFE172DC1F9EF6FA8256A57FB31046BB78844FFBE69A01FB5D390A2C095
+  ProductCode: '{648AD469-C6DA-11EE-BAE4-0CC47AC03162}'
+ManifestType: installer
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.installer.yaml:14]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+- free
+- mathematica
+- wolfram
+- wolframengine
+- wolfram-engine
+# Agreements: 
+# ReleaseNotes: 
+# ReleaseNotesUrl: 
+ManifestType: defaultLocale
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.locale.en-US.yaml:32]:
```suggestion
ManifestVersion: 1.6.0
```

File: manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.0.6 $debug=AUSU.5-1-19041-1320
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: WolframResearch.WolframEngine
+PackageVersion: 14.0.0
+DefaultLocale: en-US
+ManifestType: version
+ManifestVersion: 1.1.0
```
Comment (stephengillie) [inline manifests/w/WolframResearch/WolframEngine/14.0.0/WolframResearch.WolframEngine.yaml:8]:
```suggestion
ManifestVersion: 1.6.0
```

**Review by stephengillie — CHANGES_REQUESTED** [review]
Hi @cc-wr,

This manifest version is depreciated, but most fields are identical between versions. Usually, increasing the manifest version number is all that's needed to pass this check.

**Comment by cc-wr** [comment]
@stephengillie, this should be addressed.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**